external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -40,12 +41,40 @@ import org.apache.spark.sql.types._
 class AvroSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
   val episodesAvro = testFile("episodes.avro")
   val testAvro = testFile("test.avro")
+  val messyAvro = testFile("messy.avro")
+  val multiRecordTypeUnionAvro = testFile("multirecordtypeunion.avro")
+  val episodesSchemaFile = testFile("episodes.avsc")
+  val testSchemaFile = testFile("test.avsc")
+  val messySchemaFile = testFile("messy.avsc")
+  val multiRecordTypeUnionSchemaFile = testFile("multirecordtypeunion.avsc")
 
   override protected def beforeAll(): Unit = {
     super.beforeAll()
     spark.conf.set("spark.sql.files.maxPartitionBytes", 1024)
   }
 
+  def forceSchemaCheck(spark: SparkSession, inputPath: String, schemaFile: String): Unit = {
```

Comment:
`spark` is already available from `SharedSQLContext`. You don't need to pass it to the function